scripts/rollup/results.json

```diff
@@ -1,17 +1,17 @@
 {
-  "branch": "master",
+  "branch": "warnForKeysInFragmentsRefactor",
```

Comment:
So should be asking contributors to include this in PRs that make change to code included in bundles? If so I can add it to our PR template for now.